src/storage/mod.rs

```diff
@@ -696,7 +714,15 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
         mut ctx: Context,
         raw_keys: Vec<Vec<u8>>,
         start_ts: TimeStamp,
-    ) -> impl Future<Output = Result<(Vec<Result<KvPair>>, Statistics, PerfStatisticsDelta)>> {
+    ) -> impl Future<
+        Output = Result<(
+            Vec<Result<KvPair>>,
+            Statistics,
+            PerfStatisticsDelta,
+            KvGetLatencyStats,
```

Comment:
extract as a strcut?

Reply:
This struct is currently only used for KvGet/KvBatchGet, so it is named KvGetXxx. If we will add similar statistics to more interfaces in the future, then we may need to change KvGetXxx to a more general name.

Reply:
I mean the return type seems complicated. How about extracting `(Statistics, PerfStatisticsDelta, KvGetLatencyStats)` as a struct?

Reply:
This is reasonable~

Reply:
done [src/storage/mod.rs#L2503](https://github.com/tikv/tikv/pull/11508/files#diff-e3216d051c82e0ef881cdaeee1b11ef716ff9d72655f381da72a7828e9ca4b3cR2503)